rules/aip0233/plural_method_name_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0233
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestMethodPluralResourceName(t *testing.T) {
+	// Set up the testing permutations.
+	tests := []struct {
+		testName   string
+		methodName string
+		uriSuffix  string
+		problems   testutils.Problems
+	}{
+		{
+			testName:   "Valid-BatchCreateBooks",
+			methodName: "BatchCreateBooks",
+			uriSuffix:  "books:batchCreate",
+			problems:   testutils.Problems{},
+		},
+		{
+			testName:   "Valid-BatchCreateMen",
+			methodName: "BatchCreateMen",
+			uriSuffix:  "men:batchCreate",
+			problems:   testutils.Problems{},
+		},
+		{
+			testName:   "Invalid-SingularBus",
+			methodName: "BatchCreateBus",
+			uriSuffix:  "bus:batchCreate",
+			problems: testutils.Problems{{
+				Message:    `should be its plural form "Buses"`,
+				Suggestion: "BatchCreateBuses",
+			}},
+		},
+		{
+			testName:   "Invalid-SingularCorpPerson",
+			methodName: "BatchCreateCorpPerson",
+			uriSuffix:  "corpPerson:batchCreate",
+			problems: testutils.Problems{{
+				Message:    `should be its plural form "CorpPeople"`,
```

Comment:
Here too, this is far too long. _If_ there was no `Suggestion`, the message should just read "CorpPeople". But since there is a `Suggestion`, check that only.

Reply:
Done.